tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -211,22 +210,36 @@ spec:
 
     - name: report
       image: quay.io/enterprise-contract/ec-cli:snapshot
-      command: [cat]
-      args:
-        - "$(params.HOMEDIR)/report.yaml"
+      script: |-
+        [[ -f "$(params.HOMEDIR)/report.yaml" ]] && cat "$(params.HOMEDIR)/report.yaml" || true
 
     - name: report-json
       image: quay.io/enterprise-contract/ec-cli:snapshot
-      command: [cat]
-      args:
-        - "$(params.HOMEDIR)/report-json.json"
+      script: |-
+        [[ -f "$(params.HOMEDIR)/report-json.json" ]] && cat "$(params.HOMEDIR)/report-json.json" || true
 
     - name: summary
       image: quay.io/enterprise-contract/ec-cli:snapshot
-      command: [jq]
+      script: |-
+        [[ -f "$(results.TEST_OUTPUT.path)" ]] && jq . "$(results.TEST_OUTPUT.path)" || true
+
+    - name: info
+      image: quay.io/enterprise-contract/ec-cli:snapshot
+      command: [printf]
       args:
-        - "."
-        - "$(results.TEST_OUTPUT.path)"
+        - "----- DEBUG OUTPUT -----\n"
+
+    - name: version
+      image: quay.io/enterprise-contract/ec-cli:snapshot
+      command: [ec]
+      args:
+        - version
```

Comment:
Maybe some extra debug:

```
      - name: images
        ...
          echo "$(params.IMAGES)"

      - name: config-source
        ...
           echo "($params.POLICY_CONFIGURATION)"

      - name: config
         ...
            jq .config < "$(params.HOMEDIR)/report-json.json"

Reply:
FYI, every step we add means a new container is created.